metadata_service/api/user.py

```diff
@@ -163,11 +163,28 @@ def get(self, user_id: str) -> Iterable[Union[Mapping, int, None]]:
         :return:
         """
         try:
+            table_key = ResourceType.Table.name.lower()
+            dashboard_key = ResourceType.Dashboard.name.lower()
+            result = {
+                table_key: [],
+                dashboard_key: []
+            }  # type: Dict[str, List[Any]]
+
             resources = self.client.get_table_by_user_relation(user_email=user_id,
                                                                relation_type=UserResourceRel.own)
-            if len(resources['table']) > 0:
-                return {'table': PopularTableSchema(many=True).dump(resources['table']).data}, HTTPStatus.OK
-            return {'table': []}, HTTPStatus.OK
+            LOGGER.info('resources1 : {}'.format(resources))
```

Comment:
typo resources1

Reply:
May bad and it was part of debugging. Will remove.